packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerTicker.tsx

```diff
@@ -19,10 +19,10 @@ const containerStyles = css`
     }
 `;
 
-const countLabelStyles = css`
+const countLabelStyles = (colour?: string) => css`
     ${textSans.xsmall({ fontWeight: 'bold' })};
     font-size: 13px;
-    color: #d42d1a;
+    color: ${colour};
```

Comment:
what happens if `colour` is undefined?

Reply:
The `colour` here could have been undefined because the `stylingSettings` prop passed to the `<MomentTemplateTicker >` wasn't previously a required prop. I've updated the conditional statement in the moment banner to require the `stylingSettings` to be defined before rendering the component